platform/security/filter/security-filter-web-sso/src/main/java/org/codice/ddf/security/filter/websso/WebSSOFilter.java

```diff
@@ -123,6 +133,24 @@ public void doFilter(
     }
   }
 
+  private WebContainer getWebContainer() {
```

Comment:
✏️ Should be able to inject through blueprint and skip this direct access through osgi.

Reply:
I didn't think of that when we were looking at it. Good call.